src/visitpy/visitpy/PyQueryOverTimeAttributes.C

```diff
@@ -111,6 +111,11 @@ PyQueryOverTimeAttributes_ToString(const QueryOverTimeAttributes *atts, const ch
     else
         snprintf(tmpStr, 1000, "%suseCachedPts = 0\n", prefix);
     str += tmpStr;
+    if(atts->GetCanUseDirectDatabaseRoute())
```

Comment:
My apologies. I've been doing this so long, I forget that some steps I deem obvious, aren't really.
When you changed this field to 'internal' in the .xml file, xml2python should have been rerun. We don't want this field showing up as a settable attribute in a python script.

Reply:
That's what I figured, but the xml2atts didn't generate a Py file, and none came up when I did a file search... Have we mistakenly not created this atts file, or is this intentional?

Reply:
Scratch that. I thought xml2atts generated the Py file for us, but I guess it doesn't. I'll go ahead and run this.

Reply:
You can run xml2python from visitpy/visitpy just point it to the xml file in common/state.

Reply:
Done. Thanks!